mne/tests/test_coreg.py

```diff
@@ -443,3 +444,27 @@ def test_coreg_class_gui_match():
         [0, 0, 0, 1]]
     assert_allclose(coreg.scale, want_scale, atol=5e-4)
     assert_allclose(coreg.trans['trans'], want_trans, atol=1e-6)
+
+
+@testing.requires_testing_data
+@pytest.mark.parametrize(
+    'drop_point_kind', (FIFF.FIFFV_POINT_CARDINAL, FIFF.FIFFV_POINT_HPI,
+                        FIFF.FIFFV_POINT_EXTRA, FIFF.FIFFV_POINT_EEG))
+def test_coreg_class_init(drop_point_kind):
+    """Test that Coregistration can be instantiation with various digs."""
```

Comment:
```suggestion
    """Test that Coregistration can be instantiated with various digs."""
```

Reply:
I trust the regression tests of the backend (`mne.coreg`) so I don't think we need full coregistration regression tests starting with CTF data are needed.

I did extended the gui test to test the non-fif file loading. It works with ctf ds files and should be support anything supported by `read_raw`.